packages/compass-home/src/plugin.tsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
-import type AppRegistry from 'hadron-app-registry';
+import { type AppRegistry, AppRegistryProvider } from 'hadron-app-registry';
 import Home from './components/home';
-import AppRegistryContext from './contexts/app-registry-context';
 
 function Plugin({
-  appRegistry,
+  localAppRegistry,
   ...homeProps
 }: {
-  appRegistry: AppRegistry;
+  localAppRegistry: AppRegistry;
 } & React.ComponentProps<typeof Home>): React.ReactElement {
   return (
-    <AppRegistryContext.Provider value={appRegistry}>
+    <AppRegistryProvider localAppRegistry={localAppRegistry}>
```

Comment:
As we set up top level app registry in the compass-main, and use hooks to access it, we can skip adding one here (and also remove the localAppRegistry passed as state in activate). Otherwise we create two registry scopes one right after another, seems a bit unneccesary

Reply:
Right, done in 65fa85ffbd1b567e29676cadafae3d553ed70fcb (and removed the entire file there) :+1:

Reply:
Oops, yeah, thanks for pointing this out :+1: